test/gtest/common/test_helpers.cc

```diff
@@ -366,6 +366,22 @@ void fill_random(void *data, size_t size)
     memset((char*)data + size - remainder, 0xab, remainder);
 }
 
+void generate_random_str(std::string &s, size_t len) {
```

Comment:
i don't see where generate_random_str() is used in conn_match?

Reply:
done